.github/CODEOWNERS

```diff
@@ -1,15 +1,18 @@
 # See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
 # These owners will be the default owners for everything in
+
 # the repo. Unless a later match takes precedence,
-*                                @Jolg42 @jkomyno @prisma/client-team-ts
+
+-                                @Jolg42 @jkomyno @prisma/client-team-ts
```

Comment:
These changes look unintentional (and probably breaking), looks like the file was reformatted as Markdown.

Reply:
They totally are, thank you, fixed